### DISCUSSION TITLE
Support Svelte

### DISCUSSION THREAD

**Discussion by slidenerd** [microsoft/monaco-editor#3791] [Q&A]
Please add support for sveltejs , a short description for it would be it is the daddy of React, Angular and Vue as it has no virtual DOM, am requesting both Intellisense support and a plugin support for running monaco editor in svelte

**Comment by gufoscuro** [comment]
Strongly support this!

**Comment by froggyling** [comment]
also supported

**Comment by braebo** [comment]
I would absolutely love this

**Comment by patrickjquinn** [comment]
Any movement on this? This would be a bit of a game changer!

**Comment by SrGeneroso** [comment]
Bump this?

**Comment by effulgentsia** [comment]
Another +1. Yes, this would be great! Alternatively to Monaco maintainers adding this, is this something that could be done in userspace? If so, are there docs for how to do it? Svelte maintains [a VSCode plugin](https://marketplace.visualstudio.com/items?itemName=svelte.svelte-vscode), but is there a straightforward way to port parts of that to code for Monaco? There's https://github.com/asafamr/monaco-svelte, but that page says that it's unmaintained, and a quick scan of that project's code looks pretty daunting, doesn't look like a straightforward port of the VSCode plugin.

**Comment by tiziodcaio** [comment]
Bump this... Should we open an issue (again?) toml support is going to be implemented... IDK why this issue was converted to discussion and toml not.

**Reply by tiziodcaio** [reply]
#4830
